pom.xml

```diff
@@ -376,7 +376,7 @@
         <carbon.utils.version>2.0.2</carbon.utils.version>
 
         <!--MSF4J-->
-        <msf4j.version>2.5.1</msf4j.version>
+        <msf4j.version>2.5.2</msf4j.version>
         <msf4j.version.range>[2.5.1, 3.0.0)</msf4j.version.range>
```

Comment:
If we are depending on some feature in MSF4J v2.5.2, then `<msf4j.version.range>` has to be bumped too.
@tharindu1st  Please confirm.